File: app/src/main/java/io/github/takusan23/Kaisendon/Preference_ApplicationContext.java
package io.github.takusan23.Kaisendon;

import android.app.Application;
import android.content.Context;

public class Preference_ApplicationContext extends Application {

    private static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = this;
    }

    public static Context getContext(){
        return mContext;
    }
}

File: app/src/main/java/io/github/takusan23/Kaisendon/Preference_ApplicationContext.kt
package io.github.takusan23.Kaisendon

import android.app.Application
import android.content.Context

class Preference_ApplicationContext : Application() {

    override fun onCreate() {
        super.onCreate()
        context = this
    }

    companion object {
        var context: Context? = null
            private set
    }
}